x-pack/plugins/synthetics/server/synthetics_service/project_monitor/normalizers/common_fields.ts

```diff
@@ -67,6 +92,13 @@ export const getCustomHeartbeatId = (
   return `${monitor.id}-${projectId}-${namespace}`;
 };
 
+export const getAPMServiceName = (monitor: ProjectMonitor, defaultValue: string) => {
+  if (monitor.apmServiceName) {
+    return monitor.apmServiceName;
+  }
+  return get(monitor, ConfigKey.APM_SERVICE_NAME) || defaultValue;
+};
```

Comment:
`apmServiceName` is a new field from agent?

Reply:
Hmm... it exists on our schema, but now that I'm looking, I see that it's not actually being passed from the agent. I'll adjust this.